locales-pending/premium.ftl

```diff
@@ -26,6 +26,9 @@ toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
 
+premium-badge-label = { -brand-premium }
+premium-cta-label = Upgrade to Premium
```

Comment:
```suggestion
premium-cta-label = Upgrade to { -brand-premium }
```

I assume we want to enforce the same brand use here?

Reply:
Right — updated!